qiskit_machine_learning/utils/loss_functions/loss_functions.py

```diff
@@ -13,46 +13,95 @@
 """ Loss utilities """
 
 from abc import ABC, abstractmethod
+from typing import Union, Tuple
 import numpy as np
 
+
 from ...exceptions import QiskitMachineLearningError
 
 
 class Loss(ABC):
     """
-    Abstract base class for Loss.
+    Abstract base class for computing Loss.
+
     """
 
-    def __call__(self, predict, target):
+    def __call__(self, predict: Union[int, np.ndarray], target: Union[int, np.ndarray]) -> float:
+        """
+        Args:
+            predict: a numpy array of predicted values using the model
+            target: a numpy array of the true values
+
+        Returns:
+            a float value of the loss function
+        """
         return self.evaluate(predict, target)
 
     @abstractmethod
-    def evaluate(self, predict, target):
-        """evaluate"""
+    def evaluate(self, predict: Union[int, np.ndarray], target: Union[int, np.ndarray]) -> float:
+        """
+        An abstract method for evaluating the loss function
+
+        Args:
+            predict: a numpy array of predicted values using the model
+            target: a numpy array of the true values
+
```

Comment:
It would be good to have a Returns here to state what the float return is

Reply:
> 
> 
> It would be good to have a Returns here to state what the float return is

Added a Returns section.